packages/frontend/src/redux/actions/staking.js

```diff
@@ -6,28 +6,20 @@ import {
     ACCOUNT_HELPER_URL,
     REACT_APP_USE_TESTINGLOCKUP,
     STAKING_GAS_BASE,
+    FARMING_CLAIM_GAS,
```

Comment:
:hocho: Opportunistic reformatting in this file that changed a bunch of imports from one-per-line to multiple-per-line.  Since we prefer 1-per-line for large objects, this will result in 'flapping' next time someone else touches the file.